docs/source/api/apollo-server.md

```diff
@@ -316,3 +316,7 @@ addMockFunctionsToSchema({
    itself. Set this to false to disable. You can manually invoke 'stop()' and
    'sendReport()' on other signals if you'd like. Note that 'sendReport()'
    does not run synchronously so it cannot work usefully in an 'exit' handler.
+
+*  `disableInterval`: boolean;
```

Comment:
Do we need to expose this on `ApolloServer`? I would expect it to be a setting that particular integrations (like the Lambda one) can turn on.